<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YYDict</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon_dict2.png</normaloff>icon_dict2.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>278</width>
     <height>225</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>有道</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>百度</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button1">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
